corehq/apps/app_manager/static/app_manager/js/hq.app_data_drilldown.ng.js

```diff
@@ -64,6 +64,8 @@
         self._forms_by_app_by_module = {};
         self._case_types_by_app = {};
 
+        self._select2Test = {};
```

Comment:
just realized this was added here just for testing purposes. i think fine for now, but not a huge fan of making code more complex for testing. one other way to do this would be to use `sinon.js` and mock out the `select2` function call make sure that it gets called with the parameters you expect. http://sinonjs.org/docs/#stubs